tests/kafkatest/services/verifiable_producer.py

```diff
@@ -132,9 +133,17 @@ def start_cmd(self, node):
         cmd += " 2>> %s | tee -a %s &" % (VerifiableProducer.STDOUT_CAPTURE, VerifiableProducer.STDOUT_CAPTURE)
         return cmd
 
+    def kill_node(self, node, clean_shutdown=True, allow_fail=False):
+        if clean_shutdown:
+            sig = signal.SIGTERM
+        else:
+            sig = signal.SIGKILL
+        for pid in self.pids(node):
+            node.account.signal(pid, sig, allow_fail)
+
     def pids(self, node):
         try:
-            cmd = "ps ax | grep -i VerifiableProducer | grep java | grep -v grep | awk '{print $1}'"
+            cmd = "jps | grep -i VerifiableProducer | awk '{print $1}'"
```

Comment:
It was needed to get around this issue: https://github.com/confluentinc/ducktape/issues/125.